docs/source/theory.rst

```diff
@@ -272,9 +276,75 @@ Finally, convective heat fluxes can be applied to boundaries:
 
 where :math:`h` is the heat transfer coefficient and :math:`T_{\mathrm{ext}}` is the external temperature.
 
----------------
+Kinetic surface model
+^^^^^^^^^^^^^^^^^^^^^
+
+Modelling hydrogen retention or outgassing might require considering the kinetics of surface processes. 
+A representative example is the hydrogen uptake from a gas phase, when the energy of incident atoms/molecules is not high enough to 
+overcome the surface barrier for implantation. The general approach to account for surface kinetics :cite:`Pick1985, Hodille2017, Guterl2019, Schmid2021` consists in 
+introducing hydrogen surface species :math:`c_\mathrm{s}\,[\mathrm{m}^{-2}]`. 
+
+Evolution of hydrogen surface concentration is determined by the atomic flux balance at the surface, as sketched in the simplified energy diagram below.
+
+.. thumbnail:: images/potential_diagram.png
+    :align: center
+    :width: 800
+
+    Idealised potential energy diagram for hydrogen near a surface of an endothermic metal. Energy levels are measured from the :math:`\mathrm{H}_2` state
+
+The governing equation for surface species is:
+
+.. math::
+    :label: eq_surf_conc
+    
+    \dfrac{d c_\mathrm{s}}{d t} = J_\mathrm{bs} - J_\mathrm{sb} + J_\mathrm{vs}
+
+where :math:`J_\mathrm{bs}\,[\mathrm{m}^{-2}\,\mathrm{s}^{-1}]` is the flux of hydrogen atoms from the subsurface (bulk region just beneath the surface) onto the surface, 
+:math:`J_\mathrm{sb}\,[\mathrm{m}^{-2}\,\mathrm{s}^{-1}]` is the flux of hydrogen atoms from the surface into the subsurface, and :math:`J_\mathrm{vs}\,[\mathrm{m}^{-2}\,\mathrm{s}^{-1}]` is the net flux of hydrogen 
+atoms from the vacuum onto the surface. It worth noticing that the current model does not account for possible surface diffusion and is available only for 1D hydrogen transport simulations.
+
+The connection condition between surface and bulk domains represents the Robin boundary condition for the hydrogen transport problem.
+
+.. math::
+    :label: eq_subsurf_conc
+    
+    -D \nabla c_\mathrm{m} \cdot \mathbf{n} = \lambda_{\mathrm{IS}} \dfrac{\partial c_{\mathrm{m}}}{\partial t} + J_{\mathrm{bs}} - J_{\mathrm{sb}}
+
+where :math:`\lambda_\mathrm{IS}\,[\mathrm{m}]` is the distance between two interstitial sites in the bulk. 
+
+.. note::
+
+    At steady state and :math:`x=0`, :eq:`eq_subsurf_conc` is reduced to :math:`D\frac{\partial c_\mathrm{m}}{\partial x}=J_\mathrm{bs}-J_\mathrm{sb}`
```

Comment:
Actually that's not completely correct. Consider the domain $\Omega = [-1, 0]$ then at $x=0$ the outward normal is $+\vec{x}$.

I know that most people assume positive x values only but is it worth adding something?

Reply:
What do you think of: `At steady-state, eq.(28) for the left domain boundary is reduced to ...`

Reply:
Yes that sounds good. Maybe to be sure we can add "at the left boundary, the normal vector $\mathbf{n}$ is $-\vec{x}$ therefore ..." or something like that?

Reply:
So, maybe this will fulfill all the gaps?

At the left boundary, the normal vector $\textbf{n}$ is $-\vec{x}$. The steady-state approximation of eq. (8) at the left boundary is, therefore, $D\frac{\partial c_\mathrm{m}}{\partial x}=J_{\mathrm{bs}}-J_{\mathrm{sb}}$

Reply:
perfect!